source/content/guides/global-cdn/01-introduction.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Pantheon Global CDN
+subtitle: Introduction
+description: Improve site performance and security with Pantheon's Global CDN.
+categories: [performance]
+tags: [cache, cdn, launch, security]
+contributors: [whitneymeredith]
+layout: guide
+showtoc: true
+permalink: docs/guides/global-cdn
+anchorid: global-cdn
+---
+
+Pantheon's [Global CDN](https://pantheon.io/features/global-cdn) is a core platform offering that provides improved performance and security for customer sites. Global CDN supports caching to accelerate both static content and anonymous site pages. By serving data from cache servers all over the world, website visitors receive a response without having to wait. 
+
+## How Does Global CDN Work?
+
+Global CDN takes Pantheon's high-performance page and asset caching system, [Varnish](/cache-control#see-also), and pushes it out globally. Rather than requests coming all the way to our primary data center, we terminate HTTPS and serve pages from a location much closer to the end-user. This speeds up the time to render a web page significantly.
+
+Global CDN allows you to tap into powerful and strategically distributed CHI metro Points of Presence (POPs) around the globe. Site pages and assets are cached at these POPs. Your site can free up its PHP workers and database to process more dynamic requests by using these CHI metro POPs. Each POP can handle hundreds of thousands of requests per second, much more than a site's own PHP and database containers.
```

Comment:
I'm just starting to review this doc, but wanted to mention this.  I still think we shouldn't be specifying "CHI Metro".  CHI Metro is one point of presence of the dozens available, and the whole point of global CDN is using the site closest to the user.  I recommend validating that we should specify this with an SME.